WebApp/autoreduce_webapp/templates/run_summary.html

```diff
@@ -102,6 +102,14 @@ <h2>Reduction Job #{{ run.title }}</h2>
                                         <em>Not yet finished</em>
                                     {% endif %}
                                 </div>
+                                <div>
+                                    <strong>Software used:</strong>
+                                    {% if run.software is not null %}
```

Comment:
NITPICK/Suggestion: This will still work if we alter this slightly to  `{% if run.software is not null %}` which looks a little cleaner. This is only a personal preference so no need to change if you don't agree :)

Reply:
I believe I have written it as `{% if run.software is not null %}`. This follows the style of other conditions in the file. Do you mean something else like `{% if run.software %}`?

Reply:
sorry, yes i do XD i just blatantly copied if without actually changing it - my bad